<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Purchase</class>
 <widget class="QWidget" name="Purchase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>631</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Franklin Gothic Book</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QTableWidget" name="tableWidget_purchase">
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_orderNumber">
     <property name="font">
      <font>
       <family>Franklin Gothic Book</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Заказ №</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton_purchaseCompletion">
     <property name="text">
      <string>Завершить покупку</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_thanksForPurchase">
     <property name="font">
      <font>
       <family>Franklin Gothic Book</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Благодарим за оформление заказа!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="pushButton_backToCatalogAfterPurchase">
     <property name="text">
      <string>Перейти в каталог</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="groupBox_receivingMethod">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Выберите способ получения товара</string>
     </property>
     <widget class="QRadioButton" name="radioButton_self">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Самовывоз</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_delivery">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Доставка</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_totalPrice">
     <property name="font">
      <font>
       <family>Franklin Gothic Book</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Итоговая сумма к оплате - </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
